src/views/Home.vue

```diff
@@ -90,12 +99,83 @@
 </template>
 
 <script>
+import { mapState } from 'vuex'
+
 import UpdateNotification from '@/components/UpdateNotification'
+import { PEER_CONNECTED } from '@/store/mutation-types'
+const STORAGE_KEY = 'ambianic-pnp-settings'
 
 export default {
   name: 'Home',
   components: {
     UpdateNotification
+  },
+  computed: {
+    ...mapState({
+      peerConnectionStatus: (state) => state.pnp.peerConnectionStatus,
+      isEdgeConnected: (state) =>
+        state.pnp.peerConnectionStatus === PEER_CONNECTED
+    })
+  },
+  data () {
+    return { hasSetupSystem: false, isLoading: true }
+  },
+  created () {
+    const setupStatus = window.localStorage.getItem('hasCompletedOnboarding')
```

Comment:
@vickywane please use vuex to access local storage items. Shortcutting vuex causes issues with tracking mutations. This is mostly a problem with writes, but we should stick to best practices.

Reply:
[See here](https://github.com/ambianic/ambianic-ui/blob/6234856d507f11fa46cf08413e1292515dff52e7/src/views/Settings.vue#L329).

```
    ...mapState({
      edgePeerId: state => state.pnp.remotePeerId,
    }),
```